### ISSUE TITLE
Publish on PyPI

### ISSUE THREAD

**Issue by sneakers-the-rat** [datajoint/element-deeplabcut#20]
### Description

README says 

`pip install element-deeplabcut`

no project is to be found on pypi! https://pypi.org/project/element-deeplabcut/

plz advise if i should pin the repo or if there's some way i can help w/ packaging.

**Comment by kabilar** [comment]
Thanks for the report @sneakers-the-rat.

We just went through a development sprint.  We will be tagging and releasing these changes to pypi by the end of May.  For the interim, please install from source:

```
pip install git+https://github.com/datajoint/element-event.git
```

Install a specific commit:
```
pip install git+https://github.com/datajoint/element-event.git@<Enter commit hash>
```

**Comment by CBroz1** [comment]
Fixed by #33
